Sources/SnapshotTesting/Snapshotting/URLRequest.swift

```diff
@@ -16,4 +16,39 @@ extension Snapshotting where Value == URLRequest, Format == String {
 
     return ([method] + headers + body).joined(separator: "\n")
   }
+  
+  /// A snapshot strategy for comparing requests based on a cURL representation.
+  public static let curl = SimplySnapshotting.lines.pullback { (request: URLRequest) in
+
+    var components = ["curl"]
+
+    // HTTP Method
+    let httpMethod = request.httpMethod!
+    switch httpMethod {
+    case "GET": break
+    case "HEAD": components.append("--head")
+    default: components.append("-X \(httpMethod)")
+    }
+
+    // Headers
+    if let headers = request.allHTTPHeaderFields {
+      for (field, value) in headers where field != "Cookie" {
+        let escapedValue = value.replacingOccurrences(of: "\"", with: "\\\"")
+        components.append("-H \"\(field): \(escapedValue)\"")
```

Comment:
Whatdya think about expanding the short argument names for their longer versions?

```suggestion
        components.append("--header \"\(field): \(escapedValue)\"")
```

Since snapshots don't need to be particularly compact it might be nice to have to be intimately familiar with all of the shorthand curl arguments.